pkg/katapult/virtual_machine_group.go

```diff
@@ -0,0 +1,66 @@
+package katapult
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/augurysys/timestamp"
+)
+
+type VirtualMachineGroupsClient struct {
+	client   *apiClient
+	basePath *url.URL
+}
+
+type virtualMachineGroupsResponseBody struct {
+	VirtualMachineGroups []*VirtualMachineGroup `json:"virtual_machine_groups,omitempty"`
+}
+
+type VirtualMachineGroup struct {
```

Comment:
This is a nitpick, but all other resources have the main struct at the top of the file, and the client stuct after wards, so would you mind moving `VirtualMachineGroup` here to the top? :)